industrial_taxonomy/pipeline/glass_house/flow.py

```diff
@@ -0,0 +1,196 @@
+from typing import List
+
+from metaflow import (
+    conda_base,
+    current,
+    FlowSpec,
+    Parameter,
+    resources,
+    pip,
+    project,
+    Run,
+    step,
+    S3,
+)
+from metaflow.datatools.s3 import MetaflowS3NotFound
+
+from industrial_taxonomy.getters.glass_house import FullMatchResult
+
+try:  # Hack for type-hints on attributes
+    from pandas import DataFrame
+except ImportError:
+    pass
+
+
+@conda_base(python="3.9")  # Need >=3.8 on Batch for TypedDict of FullMatchResult
+@project(name="industrial_taxonomy")
+class JacchammerFlow(FlowSpec):
+    """Fuzzy match Companies House and Glass AI names using Jacchammer.
+
+    Attributes:
+        run_id: The origin run ID of the flow
+        names_x: Companies House company names
```

Comment:
minor: these names are not very informative. Perhaps change to `names_glass` and `names_ch`? Same for `clean_names`.

Reply:
:+1: This was a hangover from when I had a more general flow that took in arbitrary data